nmdc_runtime/api/main.py

```diff
@@ -356,10 +364,15 @@ async def get_versions():
         "\n\n"
         "Dependency versions:\n\n"
         f'nmdc-schema={version("nmdc_schema")}\n\n'
-        "<a href='https://microbiomedata.github.io/nmdc-runtime/'>Documentation</a>"
+        "<a href='https://microbiomedata.github.io/nmdc-runtime/'>Documentation</a>\n\n"
+        '<auth-action><img src="/static/ORCIDiD_icon128x128.png" height="18" width="18"/> '
+        f'<a href="https://orcid.org/oauth/authorize?client_id={ORCID_CLIENT_ID}'
+        "&response_type=code&scope=openid&"
+        f'redirect_uri={BASE_URL_EXTERNAL}/orcid_code">Login with ORCiD</a></auth-action>'
     ),
```

Comment:
Are the <auth-action> tags necessary? It looks like they're currently just rendered as plain text. I removed them and was still able to authenticate.

Reply:
not necessary. they were an experimental hook for me to toggle what was displayed based on auth status.